src/app/clusters/webrtc-transport-provider-server/webrtc-transport-provider-cluster.cpp

```diff
@@ -438,19 +440,17 @@ void WebRTCTransportProviderServer::HandleProvideOffer(HandlerContext & ctx, con
     if (req.streamUsage == StreamUsageEnum::kUnknownEnumValue)
     {
         ChipLogError(Zcl, "HandleProvideOffer: Invalid streamUsage value %u.", to_underlying(req.streamUsage));
-        ctx.mCommandHandler.AddStatus(ctx.mRequestPath, Status::ConstraintError);
-        return;
+        return Protocols::InteractionModel::Status::ConstraintError;
```

Comment:
```suggestion
        return Status::ConstraintError;
```

we seem to use Status. Could make this change throghout.